pkg/cmd/extension/command.go

```diff
@@ -63,17 +63,7 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 			}
 			return cmdutil.SilentError
 		}
-		if io.IsStdoutTTY() {
-			successStr := "Successfully"
-			if flagDryRun {
-				successStr = "Would have"
-			}
-			extensionStr := "extension"
-			if name == "" {
-				extensionStr = "extensions"
-			}
-			fmt.Fprintf(io.Out, "%s %s upgraded %s\n", cs.SuccessIcon(), successStr, extensionStr)
-		}
+
 		return nil
```

Comment:
I understand this defers to the extension manager to handle per-extension success / fail / drymode messaging 👍 

That said, **is there value still having something here**? 🤔 

```go
		if io.IsStdoutTTY() {
			fmt.Fprintf(io.Out, "%s Successfully checked extension upgrades\n", cs.SuccessIcon())
		}
```

Does this mean that extension manager should return some information back to the command to summarize it effectively?

cc: @williammartin @yermulnik

Reply:
Sure sounds good to me.

Reply:
@andyfeller Thank you for your help! I think there is value having that TTY message communicate that the extension upgrades were checked successfully. :smile: